index.js

```diff
@@ -91,6 +91,7 @@ function transformSource (runner, config, source, map, callback) {
 
   var dataBuffers = []
   child.stdout.on('data', function (data) {
+    if (config.debug) { console.log(data) }
```

Comment:
Is there a `console.debug` method? Maybe it should be used instead?

Reply:
There is only `console.log` and `console.error` for STDOUT and STDERR respectively.